components/ecoindex/scraper/scrap.py

```diff
@@ -142,3 +140,22 @@ async def get_nodes_count(self) -> int:
         svgs = await self.page.locator("//*[local-name()='svg']//*").all()
 
         return len(nodes) - len(svgs)
+    
+    def get_request_size(self, category: str, entry) -> int:
```

Comment:
**❓ question(non-blocking):** I am not sure of the behaviour here... Shouldn't this be simply:
> If entry["response"]["_transferSize"] != -1 then return it
> else return value of the "content-length" header

does this work or it is too simple ?

Reply:
Perfect!

Reply:
Hello @vvatelot ! 
I updated the method. I use the following logic : 

```
If entry["response"]["_transferSize"] != -1 then return it
elif  "content-length" header exist AND status == 206 then return value of the "content-length" header
else return response_header_size
```

The status 206 check is needed because we sometimes have HTTP Request with status 200, and an incorrect content-Length. It occurs when it's a kind of preflight response. 
For example the following URL (https://www.pascalfaure.com/hyper_relax/01_relaxation_corporelle.mp3) triggers two requests : 
- The first one gives the following : mimeType audio / transferSize -1 / status 200 / content-Length 15Mo / actual content Nothing
- The second request gives the following : mimeType audio / transferSize -1 / status 206 / content-Length 15Mo / actual content 15Mo

Reply:
OK, so in this case, it means that we will get in the end 15Mo from the second request and that's it ? BTW, as the first one is a preflight request, it is an OPTION request or also a GET request ?

Reply:
Yes for the first question and for the second one it is still a GET request